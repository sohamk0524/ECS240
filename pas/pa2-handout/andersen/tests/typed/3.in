c Test case with address-of and copy statements
p 8 8
t 1 2
t 2 2
t 3 2
t 4 1
t 5 1
t 6 0
t 7 0
t 8 0
c layer 2
s 0 1 -1 4
s 0 2 -1 5
s 0 3 0 1
s 0 3 0 2
c layer 1
s 0 4 -1 6
s 0 4 -1 7
s 0 5 0 4
s 0 5 -1 8
