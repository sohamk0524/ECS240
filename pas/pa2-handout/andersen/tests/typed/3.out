pt 1 4
pt 2 5
pt 3 4
pt 3 5
pt 4 6
pt 4 7
pt 5 6
pt 5 7
pt 5 8
